src/Synthesiser.cpp

```diff
@@ -739,6 +739,9 @@ void Synthesiser::emitCode(std::ostream& out, const RamStatement& stmt) {
         }
 
         void visitAggregate(const RamAggregate& aggregate, std::ostream& out) override {
+            // Current version is not parallel
```

Comment:
This seems like it should be an issue. If it really needs to be documented in the code, then label it with a `TODO (issue/owner):`